docs/src/literate-tutorials/linear_elasticity.jl

```diff
@@ -1,3 +1,243 @@
-# # Linear elasticity
+# # [Linear elasticity](@id tutorial-linear-elasticity)
 #
-# TBW
+# ![](linear_elasticity.png)
+#
+# *Figure 1*: Linear elastically deformed Ferrite logo.
+#
+#md # !!! tip
+#md #     This example is also available as a Jupyter notebook:
+#md #     [`heat_equation.ipynb`](@__NBVIEWER_ROOT_URL__/examples/heat_equation.ipynb).
+#-
+#
+# ## Introduction
+#
+# The classical first finite element problem to solve in solid mechanics is a linear balance
+# of momentum problem. We will use this to introduce a vector valued field, as well as the
+# [`Tensors.jl`](https://github.com/Ferrite-FEM/Tensors.jl) toolbox.
+#
+# The strong form of the balance of momentum is given by
+# ```math
+#  -\boldsymbol{\sigma} \cdot \boldsymbol{\nabla} = \boldsymbol{b}  \quad \textbf{x} \in \Omega,
```

Comment:
```suggestion
#  -\boldsymbol{\nabla} \cdot \boldsymbol{\sigma}(\boldsymbol{u}(\textbf{x})) = \boldsymbol{b}(\textbf{x})  \quad \textbf{x} \in \Omega,
```
maybe to highlight the dependencies?

Reply:
I left out the dependencies on x in order to make it more readable and easier to recognize. Do you think it's important for understanding? Perhaps an alternative is to highlight that in the variable explanation, i.e. something like "where \sigma(x) is the stress tensor" ?

Reply:
(The "x in Omega" at the end of the equation should go away though if I don't have x in the equation itself.)

Reply:
I think if someone without continuum mechanics background reads this, then he will have a hard time to follow due to the recursion. I usually prefer putting the dependence on the solution variable explictily in the equations, so it is clearer which terms are linear/nonlinear functions in the solution variables and which are independent. We can also add a bit more explanation below.

Reply:
We don't do this for any other tutorials as far as I can see, so presenting it this way is sufficiently clear, makes it more recognizable, and easier to read IMO.